packages/frontend/components/scaffold-eth/Balance.tsx

```diff
@@ -0,0 +1,81 @@
+import { BigNumberish, ethers } from "ethers";
+import { useEffect, useState } from "react";
+import { useBalance } from "wagmi";
+
+type BalanceProps = {
+  address: string;
+  price: number;
+  wrapperClasses?: string;
+};
+
+/**
+ Display balance of an address
+*/
+
+export default function Balance({ address, price, wrapperClasses }: BalanceProps) {
+  const [isEthBalance, setIsEthBalance] = useState(true);
+  const [balance, setBalance] = useState<any>();
```

Comment:
We don't need to do "any" here if we do the changes I requested on the useEffect